source/tyr/tyr/tasks.py

```diff
@@ -235,6 +235,34 @@ def import_autocomplete(files, autocomplete_instance, async=True, backup_file=Tr
 
 
 @celery.task()
+def import_in_mimir(files, instance, async=True):
```

Comment:
I'm not convinced by this, you create a new `Job` and attach it to the instance, so the job will be use in the next import no ?